monitoring/mongodb/alerts.yaml

```diff
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      count_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) > 0
+    for: 1h
+    labels:
+      severity: warning
+    annotations:
+      description: "The Mongodb instance `{{ $labels.pod }}` has been in the 'RECOVERING' state at least once in the past hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
+      summary: MongoDB node recovering for too long
+
+  - alert: MongoDbInvalidState
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state=~"6|8|10"}[1m]) > 0
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      description: "The Mongodb instance `{{ $labels.pod }}` is in the `{{ $labels.rs_state }}` state. Immediate attention is required."
+      summary: "MongoDB node in an invalid state: 6 (UNKNOWN), 8 (DOWN), 10 (REMOVED)"
+
+
+  - alert: MongoDbNodeStartup2
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="5"}[1h]) == 5
+    for: 1h
+    labels:
+      severity: warning
+    annotations:
+      description: "The Mongodb instance `{{ $labels.pod }}` is in the 'STARTUP2' state for an hour. The instance might be stuck."
+      summary: MongoDB node in STARTUP2 state for too long
+
+  - alert: MongoDbNodeNotSynced
+    expr: |
+      sum by (pod) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", member_state="SECONDARY"}) != (${replicas} - 1)
+    for: 1m
+    labels:
+      severity: critical
+    annotations:
+      description: "The MongoDB instance `{{ $labels.pod }}` is out of the replica set. It does no longer receive any data and must be added back to the cluster to avoid performance and storage problems."
+      summary: MongoDB node not in replica set
```

Comment:
* the `pod` would be different for each time serie (it is `${service}.*`), so the `sum` not compare to ${replicas}. To do this aggregation, we need to aggregate on a label which is hase a different value each "group" of pods (mongo-shard$X, mongo-cfgsvr) but also the same value for every pod in the group
* since we are doing an alert on the aggregate, we will not have a single "pod", and cannot point to the node in the bad state.

Reply:
done here : https://github.com/scality/Zenko/pull/2142/commits/3d767a04f769852075e490521342664328265294#diff-237b96ceb855bab52e3f81d4a49e5e17e07fc7c1102d3dec5b30e461fee6fc69R176